Tasks/Common/sonarqube-common/sonarqube-common.ts

```diff
@@ -196,6 +201,18 @@ function getOrCreateSonarQubeStagingDirectory(): string {
     return sqStagingDir;
 }
 
+// Creates a string containing Markdown of a link to the SonarQube dashboard for this project.
```

Comment:
Unrelated change: tidying up the build summary code in preparation for the next ticket.